core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -2917,34 +2925,33 @@ class GroupCoordinatorTest {
 
   @Test
   def testFetchAllOffsets(): Unit = {
-    val tp1 = new TopicPartition("topic", 0)
-    val tp2 = new TopicPartition("topic", 1)
-    val tp3 = new TopicPartition("other-topic", 0)
     val offset1 = offsetAndMetadata(15)
     val offset2 = offsetAndMetadata(16)
     val offset3 = offsetAndMetadata(17)
+    val (topicId, otherTopicId) = (Uuid.randomUuid(), Uuid.randomUuid())
+    val tip1 = new TopicIdPartition(topicId, 0, "topic")
+    val tip2 = new TopicIdPartition(topicId, 1, "topic")
+    val tip3 = new TopicIdPartition(otherTopicId, 0, "other-topic")
```

Comment:
nit: Could we keep this block first as it was? Moreover, I would define it as follow:
```
    val tip1 = new TopicIdPartition(Uuid.randomUuid(), 0, "topic")
    val tip2 = new TopicIdPartition(tip1.topic(), 1, "topic")
    val tip3 = new TopicIdPartition(tip1, 0, "other-topic")
```

Reply:
Thanks, applied the same pattern in few other places too.